inc/Utils.php

```diff
@@ -690,6 +690,12 @@ public static function get_document_kit_id( $post_id ) {
 		}
 
 		$document = Plugin::elementor()->documents->get_doc_for_frontend( $post_id );
+
+		// Check if document exists.
+		if ( ! $document ) {
+			return false;
+		}
```

Comment:
Looks like the parent function isn't used any longer. Is there any reason to keep it?

Reply:
Good catch, looks like this function's `get_document_kit_id ` usage was scraped the first time it was created and I missed removing the helper function.
Removed it!